consensus-types/interfaces/beacon_block.go

```diff
@@ -132,6 +133,9 @@ type ExecutionData interface {
 	PbCapella() (*enginev1.ExecutionPayloadCapella, error)
 	PbBellatrix() (*enginev1.ExecutionPayload, error)
 	PbDeneb() (*enginev1.ExecutionPayloadDeneb, error)
+	PbElectra() (*enginev1.ExecutionPayloadElectra, error)
 	ValueInWei() (math.Wei, error)
 	ValueInGwei() (uint64, error)
+	DepositReceipts() ([]*enginev1.DepositReceipt, error)
```

Comment:
DepositReceiptsRoot() ([]byte, error) and WithdrawalRequestsRoot() ([]byte, error) can also be added here

Reply:
Good idea

Reply:
Actually, are we sure we need it?